src/composables/use-content-type.js

```diff
@@ -0,0 +1,34 @@
+import { computed, ref, useContext } from '@nuxtjs/composition-api'
+import { supportedContentTypes } from '~/constants/media'
+import { SEARCH } from '~/constants/store-modules'
+import { UPDATE_QUERY } from '~/constants/action-types'
+import allIcon from '~/assets/icons/all-content.svg'
+import audioIcon from '~/assets/icons/audio-content.svg'
+import imageIcon from '~/assets/icons/image-content.svg'
+
+const icons = {
+  all: allIcon,
+  audio: audioIcon,
+  image: imageIcon,
+}
+export default function useContentType() {
```

Comment:
I really like this way of encapsulating parts of Vuex into a composable. It feels really clean and creates a much nicer API for the components to work with. As a bonus I suspect this would make it easier to switch to pinia in the future.